gitpod-io/website
# Filter in product (VS Code) changelog by a custom front-matter property (distinguish ad-hoc vs roll-up

## Description

Filter in product (VS Code) changelog by a custom front-matter property (distinguish ad-hoc vs roll-up

## Related Issue(s)



fix https://github.com/gitpod-io/gitpod/issues/16897

## How to test



Check /changelog/latest endpoint, play with tag properties from sources to see what it returns. There is some caching, you have to do force reload.

## Release Notes



```release-note
NONE
```
